physics/flake.F90

```diff
@@ -2466,18 +2472,37 @@ SUBROUTINE SfcFlx_momsenlat ( height_u, height_tq, fetch,                &
 ELSE                        ! Convection 
   psi_u = (1.0-c_MO_t_conv*R_z*ZoL)**c_MO_t_exp
   psi_t = (1.0-c_MO_t_conv*R_z*ZoL*MIN(z0t_sf/height_tq, 1.0))**c_MO_t_exp
-  psi_t = 2.0*LOG((1.0+psi_t)/(1.0+psi_u))
+!  psi_t = 2.0*LOG((1.0+psi_t)/(1.0+psi_u))
+  psi_t = abs(2.0*LOG((1.0+psi_t)/(1.0+psi_u)))
```

Comment:
The flake code is not my work, so I'm wary of touching it.

Reply:
That would be quite an extensive change to the file, so the flake developers should fix that, not me. It belongs in another PR, since there are no flake developers right now.